bedevere/bpo.py

```diff
@@ -37,6 +44,17 @@
         else:
             status = FAILURE_STATUS
     else:
+        if "body" in event.data["pull_request"]:
+            body = event.data["pull_request"]["body"]
+            if CLOSING_TAG not in body:
+                issue_number = issue_number_found.group("issue")
+                BPO_MSG = f"""{body}\n
+<!-- {TAG_NAME}: bpo-{issue_number} -->
```

Comment:
To make this more readable, you can indent it and then use [`textwrap.dedent()`](https://docs.python.org/3.6/library/textwrap.html#textwrap.dedent) to strip the leading whitespace.